Formula/liboping.rb

```diff
@@ -15,6 +15,11 @@ class Liboping < Formula
 
   uses_from_macos "ncurses"
 
+  patch do
+    url "https://github.com/cron2/liboping/commit/b5c5091d0214dd2c6cc504c02e2f78907abdee00.diff?full_index=1"
```

Comment:
It's better to use a commit not from a fork. but from PR:

```suggestion
    url "https://github.com/octo/liboping/pull/55/commits/b5c5091d0214dd2c6cc504c02e2f78907abdee00.patch?full_index=1
```
(also you'll need to update sha256 accordingly)

Reply:
Sure thing. Will make the changes.